src/languages/swift/index.ts

```diff
@@ -0,0 +1,149 @@
+import {
+  functionCall,
+  id,
+  indexCall,
+  methodCall,
+  int,
+  polygolfOp,
+} from "../../IR";
+import { Language } from "../../common/Language";
+
+import emitProgram from "./emit";
+import { mapOps, mapPrecedenceOps, useIndexCalls } from "../../plugins/ops";
+import { addVarDeclarations } from "../nim/plugins";
+
+import { addImports } from "./plugins";
+import { renameIdents } from "../../plugins/idents";
+import { evalStaticExpr, golfStringListLiteral } from "../../plugins/static";
+import { addMutatingBinaryOp, flipBinaryOps } from "../../plugins/binaryOps";
+import { golfLastPrint } from "../../plugins/print";
+
+const swiftLanguage: Language = {
+  name: "Swift",
+  extension: "swift",
+  emitter: emitProgram,
+  golfPlugins: [
+    flipBinaryOps,
+    golfStringListLiteral,
+    evalStaticExpr,
+    golfLastPrint(),
+  ],
+  emitPlugins: [useIndexCalls()],
+  finalEmitPlugins: [
+    mapOps([
+      [
+        "argv_get",
+        (x) =>
+          indexCall(
+            id("CommandLine.arguments", true),
+            polygolfOp("add", x[0], int(1n)),
+            "argv_get",
+            true
+          ),
+      ],
+    ]),
+    mapOps([
+      [
+        "text_get_byte",
+        (x) =>
+          functionCall(
+            [
+              indexCall(
+                functionCall([methodCall(x[0], [], "utf8")], "Array"),
+                x[1]
+              ),
+            ],
+            "Int"
+          ),
+      ],
+      [
+        "text_get_char",
+        (x) =>
+          functionCall(
+            [indexCall(functionCall([x[0]], "Array"), x[1])],
+            "String"
+          ),
+      ],
+      [
+        "byte_to_char",
+        (x) => functionCall([functionCall([x[0]], "UnicodeScalar")], "String"),
+      ],
+      ["text_length", (x) => methodCall(x[0], [], "count")],
+      // ["text_length_chars", (x) => methodCall(x[0], [], "count")],
+      // ["text_length_bytes", (x) => methodCall(methodCall(x[0], [], "utf8"), [], "count")]
```

Comment:
`text_length_chars` and `text_length_chars`: intended to be missing?

Reply:
Yes, I preemptively wrote in behavior for ops that should be added alongside #71